Formula/lib/libpipewire.rb

```diff
@@ -0,0 +1,64 @@
+class Libpipewire < Formula
```

Comment:
Should just be named `pipewire` as it is installing everything including executables. `libpipewire` is only used in repositories that support split packaging, which is difficult to do in Homebrew.